SEC/Debt/AssetBackedSecurities/PoolBackedSecurities.rdf

```diff
@@ -109,21 +93,25 @@
+		<skos:definition xml:lang="en">structured finance that offers investors exposure to chosen underlying assets using various approaches and asymmetric pay-off profiles</skos:definition>
 		<fibo-fnd-utl-av:adaptedFrom>ISO 10962, Securities and related financial instruments - Classification of Financial Instruments (CFI code), Fourth edition, 2019-10-01.</fibo-fnd-utl-av:adaptedFrom>
 		<fibo-fnd-utl-av:explanatoryNote xml:lang="en">There are one or more reference entities underlying the product. Redemption is made at least in the amount of the conditional capital protection at maturity, provided that no credit event by the reference entity has occurred. Conditional capital protection only applies to the nominal amount and not to the purchase price. The general functioning of a capital guaranteed structured instrument is as follows: the notional amount is split into a zero bond, that will deliver the capital guarantee at maturity, and the difference between the zero bond&apos;s value (= present value of the guarantee level at maturity) and the notional amount is used for structuring the performance component with options which deliver the agreed pay-off profile of the structured instrument.</fibo-fnd-utl-av:explanatoryNote>
 		<fibo-fnd-utl-av:synonym xml:lang="en">capital protected note</fibo-fnd-utl-av:synonym>
 	</owl:Class>
 	
-	<owl:Class rdf:about="&fibo-sec-dbt-ab-pbs;SecuritizedDebtPool">
-		<rdfs:subClassOf rdf:resource="&fibo-sec-sec-pls;DebtPool"/>
-		<rdfs:label xml:lang="en">securitized debt pool</rdfs:label>
-		<skos:definition xml:lang="en">pool of individual debt instruments (such as loans or mortgages) that that have been securitized through the issue of some security</skos:definition>
-		<skos:editorialNote xml:lang="en">We may not need this, but if we keep it we need to link it to the instruments that depend on it (see property &apos;backs&apos;, below).</skos:editorialNote>
-	</owl:Class>
-	
 	<owl:Class rdf:about="&fibo-sec-dbt-ab-pbs;StructuredFinanceInstrument">
 		<rdfs:subClassOf rdf:resource="&fibo-sec-dbt-ab-pbs;PoolBackedSecurity"/>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fnd-rel-rel;isIssuedBy"/>
+				<owl:someValuesFrom>
+					<owl:Restriction>
+						<owl:onProperty rdf:resource="&fibo-fnd-pty-rl;isPlayedBy"/>
+						<owl:someValuesFrom rdf:resource="&fibo-sec-sec-pls;SpecialPurposeVehicle"/>
+					</owl:Restriction>
+				</owl:someValuesFrom>
+			</owl:Restriction>
+		</rdfs:subClassOf>
 		<rdfs:label xml:lang="en">structured finance instrument</rdfs:label>
 		<rdfs:seeAlso rdf:resource="https://www.bis.org/publ/cgfs23mitchell.pdf"/>
 		<skos:definition xml:lang="en">pool-backed security wherein the risk associated with the pool has been uncoupled from the risk associated with the originating institution, typically through a special purpose vehicle</skos:definition>
```

Comment:
"typically" is wrong with respect to the new someValuesFrom restriction

Reply:
@rivettp - removed 'typically'